meta-resin-common/recipes-containers/balena/balena_git.bb

```diff
@@ -116,50 +104,54 @@ do_compile() {
 }
 
 SYSTEMD_PACKAGES = "${PN}"
-SYSTEMD_SERVICE_${PN} = "docker.service docker-host.service var-lib-docker.mount"
+SYSTEMD_SERVICE_${PN} = "balena.service balena-host.service var-lib-docker.mount"
 
 do_install() {
   mkdir -p ${D}/${bindir}
-  install -m 0755 ${S}/bundles/${DOCKER_VERSION}/dynbinary-rce-docker/rce-docker ${D}/${bindir}/rce-docker
+  install -m 0755 ${S}/bundles/${BALENA_VERSION}/dynbinary-balena/balena ${D}/${bindir}/balena
   install -d ${D}/boot
   install -m 0755 ${S}/cmd/mobynit/mobynit ${D}/boot/init
   echo ${DOCKER_STORAGE} > ${D}/boot/storage-driver
```

Comment:
Can we rename `DOCKER_STORAGE` to `BALENA_STORAGE` too?

Reply:
@petrosagg I avoided for now because there are board specific layers witch already use it. I would probably leave it for a next PR.

Reply:
Actually it will be included in the next iteration.